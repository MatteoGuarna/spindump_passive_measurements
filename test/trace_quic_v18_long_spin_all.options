--report-spin-flips --report-spins
