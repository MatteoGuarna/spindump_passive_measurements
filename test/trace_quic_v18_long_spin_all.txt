Like trace_quic_v18_long_spin, but with full reporting of spin values
