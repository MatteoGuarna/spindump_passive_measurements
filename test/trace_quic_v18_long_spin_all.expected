QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-0a674f45296cdd8c (55782:4433) at 1553417138023042 new starting packets 0 0 bytes 0 0
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138038065 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138038065 measurement up right 15023 packets 1 0 bytes 1228 0 bandwidth 1228 0
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138038257 spinvalue up 0 responder packets 1 1 bytes 1228 424 bandwidth 1228 424
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138039083 spinvalue up 0 responder packets 1 2 bytes 1228 754 bandwidth 1228 754
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138044396 spinvalue up 0 responder packets 1 3 bytes 1228 2034 bandwidth 1228 2034
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138049019 spinvalue up 1 initiator packets 2 4 bytes 1372 3314 bandwidth 1372 3314
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138057583 spinvalue up 0 responder packets 3 4 bytes 1457 3314 bandwidth 1457 3314
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138060015 spinvalue up 0 responder packets 3 5 bytes 1457 4594 bandwidth 1457 4594
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138061238 spinvalue up 1 initiator packets 3 6 bytes 1457 5874 bandwidth 1457 5874
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138062653 spinvalue up 0 responder packets 4 6 bytes 1521 5874 bandwidth 1521 5874
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138062655 spinvalue up 0 responder packets 4 7 bytes 1521 7154 bandwidth 1521 7154
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138062656 spinvalue up 0 responder packets 4 8 bytes 1521 8434 bandwidth 1521 8434
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138062657 spinvalue up 0 responder packets 4 9 bytes 1521 9714 bandwidth 1521 9714
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138062842 spinvalue up 0 responder packets 4 10 bytes 1521 10994 bandwidth 1521 10994
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138062844 spinvalue up 1 responder packets 4 12 bytes 1521 12347 bandwidth 1521 12347
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138062844 spinflip up 0-1 responder packets 4 12 bytes 1521 12347 bandwidth 1521 12347
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138063011 spinvalue up 1 responder packets 4 13 bytes 1521 13627 bandwidth 1521 13627
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138063013 spinvalue up 1 responder packets 4 14 bytes 1521 14907 bandwidth 1521 14907
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138063014 spinvalue up 1 responder packets 4 15 bytes 1521 16187 bandwidth 1521 16187
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138063192 spinvalue up 1 responder packets 4 16 bytes 1521 17467 bandwidth 1521 17467
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138066101 spinvalue up 1 initiator packets 4 17 bytes 1521 18747 bandwidth 1521 18747
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138072533 spinvalue up 0 initiator packets 5 17 bytes 1583 18747 bandwidth 1583 18747
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138072533 spinflip up 1-0 initiator packets 5 17 bytes 1583 18747 bandwidth 1583 18747
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138072533 measurement up left 9689 packets 5 17 bytes 1583 18747 bandwidth 1583 18747
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138074903 spinvalue up 1 responder packets 6 17 bytes 1645 18747 bandwidth 1645 18747
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138076047 spinvalue up 1 responder packets 6 18 bytes 1645 20027 bandwidth 1645 20027
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138076049 spinvalue up 1 responder packets 6 19 bytes 1645 21307 bandwidth 1645 21307
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138076050 spinvalue up 1 responder packets 6 20 bytes 1645 22587 bandwidth 1645 22587
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138080313 spinvalue up 1 responder packets 6 21 bytes 1645 23867 bandwidth 1645 23867
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138080317 spinvalue up 1 responder packets 6 22 bytes 1645 25147 bandwidth 1645 25147
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138080318 spinvalue up 1 responder packets 6 23 bytes 1645 26427 bandwidth 1645 26427
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138080332 spinvalue up 0 initiator packets 6 24 bytes 1645 27707 bandwidth 1645 27707
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138080508 spinvalue up 1 responder packets 7 24 bytes 1730 27707 bandwidth 1730 27707
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138080511 spinvalue up 1 responder packets 7 25 bytes 1730 28987 bandwidth 1730 28987
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138080513 spinvalue up 1 responder packets 7 26 bytes 1730 30267 bandwidth 1730 30267
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138080540 spinvalue up 1 responder packets 7 27 bytes 1730 31547 bandwidth 1730 31547
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138080679 spinvalue up 1 responder packets 7 28 bytes 1730 32827 bandwidth 1730 32827
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138085162 spinvalue up 0 initiator packets 7 29 bytes 1730 34107 bandwidth 1730 34107
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138090745 spinvalue up 0 initiator packets 8 29 bytes 1792 34107 bandwidth 1792 34107
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138091401 spinvalue up 0 responder packets 9 29 bytes 1854 34107 bandwidth 1854 34107
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138091401 spinflip up 1-0 responder packets 9 29 bytes 1854 34107 bandwidth 1854 34107
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138091401 measurement up right 18868 packets 9 29 bytes 1854 34107 bandwidth 1854 34107
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138091401 measurement up full (responder) 28557 packets 9 29 bytes 1854 34107 bandwidth 1854 34107
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138096176 spinvalue up 0 responder packets 9 30 bytes 1854 34192 bandwidth 1854 34192
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138096762 spinvalue up 0 responder packets 9 31 bytes 1854 35472 bandwidth 1854 35472
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138096765 spinvalue up 0 responder packets 9 32 bytes 1854 36752 bandwidth 1854 36752
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138096766 spinvalue up 0 responder packets 9 33 bytes 1854 38032 bandwidth 1854 38032
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138097320 spinvalue up 1 initiator packets 9 34 bytes 1854 39312 bandwidth 1854 39312
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138097320 spinflip up 0-1 initiator packets 9 34 bytes 1854 39312 bandwidth 1854 39312
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138097320 measurement up left 5919 packets 9 34 bytes 1854 39312 bandwidth 1854 39312
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138097320 measurement up full (initiator) 24787 packets 9 34 bytes 1854 39312 bandwidth 1854 39312
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138099572 spinvalue up 1 initiator packets 10 34 bytes 1916 39312 bandwidth 1916 39312
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138102323 spinvalue up 0 responder packets 11 34 bytes 1984 39312 bandwidth 1984 39312
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138104041 spinvalue up 0 responder packets 11 35 bytes 1984 40592 bandwidth 1984 40592
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138104043 spinvalue up 0 responder packets 11 36 bytes 1984 41872 bandwidth 1984 41872
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138104044 spinvalue up 0 responder packets 11 37 bytes 1984 43152 bandwidth 1984 43152
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138104045 spinvalue up 0 responder packets 11 38 bytes 1984 44432 bandwidth 1984 44432
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138107677 spinvalue up 1 initiator packets 11 39 bytes 1984 45712 bandwidth 1984 45712
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138110811 spinvalue up 1 responder packets 12 39 bytes 2046 45712 bandwidth 2046 45712
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138110811 spinflip up 0-1 responder packets 12 39 bytes 2046 45712 bandwidth 2046 45712
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138110811 measurement up right 13491 packets 12 39 bytes 2046 45712 bandwidth 2046 45712
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138110811 measurement up full (responder) 19410 packets 12 39 bytes 2046 45712 bandwidth 2046 45712
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138111266 spinvalue up 1 responder packets 12 40 bytes 2046 46992 bandwidth 2046 46992
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138111269 spinvalue up 1 responder packets 12 41 bytes 2046 48272 bandwidth 2046 48272
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138111270 spinvalue up 1 responder packets 12 42 bytes 2046 49552 bandwidth 2046 49552
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138114491 spinvalue up 0 initiator packets 12 43 bytes 2046 50832 bandwidth 2046 50832
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138114491 spinflip up 1-0 initiator packets 12 43 bytes 2046 50832 bandwidth 2046 50832
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138114491 measurement up left 3680 packets 12 43 bytes 2046 50832 bandwidth 2046 50832
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138114491 measurement up full (initiator) 17171 packets 12 43 bytes 2046 50832 bandwidth 2046 50832
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138119346 spinvalue up 1 responder packets 13 43 bytes 2108 50832 bandwidth 2108 50832
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138121755 spinvalue up 0 initiator packets 13 44 bytes 2108 51720 bandwidth 2108 51720
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138133019 spinvalue up 0 responder packets 14 44 bytes 2175 51720 bandwidth 2175 51720
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138133019 spinflip up 1-0 responder packets 14 44 bytes 2175 51720 bandwidth 2175 51720
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138133019 measurement up right 18528 packets 14 44 bytes 2175 51720 bandwidth 2175 51720
QUIC 31.133.149.35 <-> 52.58.13.57 84eb460f-851aedc3018c73b5 (55782:4433) at 1553417138133019 measurement up full (responder) 22208 packets 14 44 bytes 2175 51720 bandwidth 2175 51720
